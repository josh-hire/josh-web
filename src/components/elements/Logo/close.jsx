import React from 'react';

export const IconClose = () => {
  return (
    <svg height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
      <path d="M18.2812 5.71875C18.6696 6.10708 18.6696 6.73667 18.2812 7.125L13.4062 12L18.2812 16.875C18.6696 17.2633 18.6696 17.8929 18.2812 18.2812V18.2812C17.8929 18.6696 17.2633 18.6696 16.875 18.2812L12 13.4062L7.125 18.2812C6.73667 18.6696 6.10708 18.6696 5.71875 18.2812V18.2812C5.33042 17.8929 5.33042 17.2633 5.71875 16.875L10.5938 12L5.71875 7.125C5.33042 6.73667 5.33042 6.10708 5.71875 5.71875V5.71875C6.10708 5.33042 6.73667 5.33042 7.125 5.71875L12 10.5938L16.875 5.71875C17.2633 5.33042 17.8929 5.33042 18.2812 5.71875V5.71875Z" />
    </svg>
  );
};
