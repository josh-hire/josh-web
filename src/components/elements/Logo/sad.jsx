import React from 'react';

export const IconSad = () => {
  return (
    <svg height="48" viewBox="0 0 48 48" width="48" xmlns="http://www.w3.org/2000/svg">
      <path d="M24.0031 3.99805C35.0505 3.99805 44.0062 12.9537 44.0062 24.0011C44.0062 35.0486 35.0505 44.0042 24.0031 44.0042C12.9557 44.0042 4 35.0486 4 24.0011C4 12.9537 12.9557 3.99805 24.0031 3.99805ZM24.0031 26.9961C20.7334 26.9961 17.669 28.3117 15.4266 30.6017C14.847 31.1936 14.857 32.1433 15.4488 32.7229C16.0407 33.3025 16.9904 33.2925 17.5701 32.7006C19.2538 30.9812 21.5483 29.9961 24.0031 29.9961C26.4525 29.9961 28.7424 30.9768 30.4253 32.6896C31.0059 33.2805 31.9557 33.2888 32.5466 32.7082C33.1375 32.1276 33.1458 31.1779 32.5652 30.587C30.3238 28.3059 27.2656 26.9961 24.0031 26.9961ZM18.004 17.5024C16.6241 17.5024 15.5054 18.6211 15.5054 20.001C15.5054 21.3809 16.6241 22.4996 18.004 22.4996C19.3839 22.4996 20.5026 21.3809 20.5026 20.001C20.5026 18.6211 19.3839 17.5024 18.004 17.5024ZM30.004 17.5024C28.6241 17.5024 27.5054 18.6211 27.5054 20.001C27.5054 21.3809 28.6241 22.4996 30.004 22.4996C31.3839 22.4996 32.5026 21.3809 32.5026 20.001C32.5026 18.6211 31.3839 17.5024 30.004 17.5024Z" />
    </svg>
  );
};
