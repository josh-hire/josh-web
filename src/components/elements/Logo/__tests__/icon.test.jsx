import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { IconClose } from '../close';
import { IconSad } from '../sad';

describe('src/components/elements/Logo', ()=>{
  test('IconClose',()=>{
    const { asFragment } = render(<IconClose />);
    expect(asFragment()).toMatchSnapshot();
  });
  test('IconSad',()=>{
    const { asFragment } = render(<IconSad />);
    expect(asFragment()).toMatchSnapshot();
  });
});
