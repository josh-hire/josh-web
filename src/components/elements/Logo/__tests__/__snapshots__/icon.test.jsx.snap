// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/Logo IconClose 1`] = `
<DocumentFragment>
  <svg
    height="24"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M18.2812 5.71875C18.6696 6.10708 18.6696 6.73667 18.2812 7.125L13.4062 12L18.2812 16.875C18.6696 17.2633 18.6696 17.8929 18.2812 18.2812V18.2812C17.8929 18.6696 17.2633 18.6696 16.875 18.2812L12 13.4062L7.125 18.2812C6.73667 18.6696 6.10708 18.6696 5.71875 18.2812V18.2812C5.33042 17.8929 5.33042 17.2633 5.71875 16.875L10.5938 12L5.71875 7.125C5.33042 6.73667 5.33042 6.10708 5.71875 5.71875V5.71875C6.10708 5.33042 6.73667 5.33042 7.125 5.71875L12 10.5938L16.875 5.71875C17.2633 5.33042 17.8929 5.33042 18.2812 5.71875V5.71875Z"
    />
  </svg>
</DocumentFragment>
`;

exports[`src/components/elements/Logo IconSad 1`] = `
<DocumentFragment>
  <svg
    height="48"
    viewBox="0 0 48 48"
    width="48"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M24.0031 3.99805C35.0505 3.99805 44.0062 12.9537 44.0062 24.0011C44.0062 35.0486 35.0505 44.0042 24.0031 44.0042C12.9557 44.0042 4 35.0486 4 24.0011C4 12.9537 12.9557 3.99805 24.0031 3.99805ZM24.0031 26.9961C20.7334 26.9961 17.669 28.3117 15.4266 30.6017C14.847 31.1936 14.857 32.1433 15.4488 32.7229C16.0407 33.3025 16.9904 33.2925 17.5701 32.7006C19.2538 30.9812 21.5483 29.9961 24.0031 29.9961C26.4525 29.9961 28.7424 30.9768 30.4253 32.6896C31.0059 33.2805 31.9557 33.2888 32.5466 32.7082C33.1375 32.1276 33.1458 31.1779 32.5652 30.587C30.3238 28.3059 27.2656 26.9961 24.0031 26.9961ZM18.004 17.5024C16.6241 17.5024 15.5054 18.6211 15.5054 20.001C15.5054 21.3809 16.6241 22.4996 18.004 22.4996C19.3839 22.4996 20.5026 21.3809 20.5026 20.001C20.5026 18.6211 19.3839 17.5024 18.004 17.5024ZM30.004 17.5024C28.6241 17.5024 27.5054 18.6211 27.5054 20.001C27.5054 21.3809 28.6241 22.4996 30.004 22.4996C31.3839 22.4996 32.5026 21.3809 32.5026 20.001C32.5026 18.6211 31.3839 17.5024 30.004 17.5024Z"
    />
  </svg>
</DocumentFragment>
`;
